node/src/utils/pid_file.rs

```diff
@@ -0,0 +1,246 @@
+//! PidFile utility type.
```

Comment:
I think we should avoid re-introducing the PidFile as it's overkill for this task.  We only need an empty file named `force_resync` (without the `.pid` suffix).

Reply:
Done.